pennylane_lightning/lightning_qubit/_state_vector.py

```diff
@@ -143,6 +156,39 @@ def _apply_lightning_controlled(self, operation, adjoint):
                 adjoint,
             )
 
+    def _apply_lightning_controlled_sparse(self, operation):
+        """Apply an arbitrary controlled operation to the state tensor.
+
+        Args:
+            operation (~pennylane.operation.Operation): controlled operation to apply
+        Returns:
+            None
+        """
+        state = self.state_vector
+
+        if isinstance(operation.base, Adjoint):
+            base_operation = operation.base.base
+            adjoint = not adjoint
```

Comment:
What is `adjoint` here? I know `_apply_lightning_controlled` has `adjoint` arg but here it is very confusing to me. Is it defined globally?

Reply:
this is wrong, thank you for catching it.